data/avro-schema/src/main/resources/avro/net/corda/data/virtualnode/VirtualNodeRequest.avsc

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "record",
+  "name": "VirtualNodeRequest",
+  "doc": "Make a request for some virtual node type. The action is determined by the type of the {@link VirtualNodeRequest#request} payload.  See also: {@link VirtualNodeResponse}, {@link CreateVirtualNode}, {@link VirtualNodeInfo}, {@link UpdateVirtualNodeState}",
+  "namespace": "net.corda.data.virtualnode",
+  "fields": [
+    {
+      "name": "timestamp",
+      "type": {
+        "type": "long",
+        "logicalType": "timestamp-millis"
+      },
+      "doc": "Time ({@link java.time.Instant}) in milliseconds when the record was created."
+    },
+    {
+      "name": "request",
+      "doc": "The 'request' that we wish to make to the persistence API.",
+      "type": [
+        "net.corda.data.virtualnode.VirtualNodeCreate",
+        "net.corda.data.virtualnode.VirtualNodeInfo"
```

Comment:
I kind of understood why we would include `VirtualNodeInfo` into the response.
But here when we are including `VirtualNodeInfo` into the request it is not immediately obvious to me what are we requesting.
I would suggest we remove `VirtualNodeInfo` from request. I think we agreed to later on add `VirtualNodeStateUpdate` into the request as a separate PR.

Reply:
We already have `VirtualNodeInfoProcessor` which gets vnode info from Kafka and loads into a cache, so we don't need a separate request making round trips to the database to get vnode info, afaik

Reply:
I agree with @conalsmith-r3 except that I would not use `VirtualNodeUpdateRequest` as it implies that anything on vNode can be updated. For our case (in the separate PR) I would call it `VirtualNodeStateChangeRequest`.

Reply:
Yeah a `VirtualNodeStateChangeRequest` is good.